src/qt/forms/researcherwizardpoolpage.ui

```diff
@@ -186,7 +186,7 @@
    <item>
     <widget class="QLabel" name="addressParagraphLabel">
      <property name="text">
-      <string>When signing up, the pool may ask for a payment address to send earnings to. Press the button to generate an address:</string>
+      <string>While signing up, the pool may ask for a payment address to send earnings to. Press the button below to generate an address.</string>
```

Comment:
```suggestion
      <string>As you sign up, the pool may ask for a payment address to send earnings to. Press the button below to generate an address.</string>
```

Reply:
Better.

Reply:
```suggestion
      <string>As you sign up, the pool will require a payment address to send earnings to. Press the button below to generate an address.</string>
```
Perhaps? The user needs to provide an address if they want to recieve GRC.

Reply:
The Arikado pool requires an address on the sign up form, but grcpool lets a user put one in at leisure. The address is effectively required in either case but not strictly when signing up. I left the language fuzzy to indicate to the user that they may need to figure that part out.

Reply:
Good point.